### ISSUE TITLE
reindex filesets in staging and prod

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/adventist-dl#367]

**Comment by jillpe** [comment]
Dependent on #345

**Comment by jillpe** [comment]
To Do:
- We reindexed all the works, but need to reindex all of the filesets too

**Comment by jillpe** [comment]
low priority
